### PR TITLE
Pass v-icon name as prop instead of slot

### PR DESCRIPTION
**Pull request by peterhauke**
## Description

docs: pass icon name as a prop and not as a slot - as recommended in 
[Vuetify docs on icon usage]
(https://vuetifyjs.com/en/features/icon-fonts/#usage)

docs; remove closing v-icon tags which are not necessary now

### CODE REVIEW

File: packages/docs/src/examples/v-icon/prop-color.vue [hunk 1]
```diff
@@ -3,43 +3,32 @@
     <v-icon
       size="large"
       color="green-darken-2"
-    >
-      mdi-domain
-    </v-icon>
-
+      icon = "mdi-domain"
```
Comment (MajesticPotatoe) [inline packages/docs/src/examples/v-icon/prop-color.vue:6]:
```suggestion
      icon="mdi-domain"
```

**Review by MajesticPotatoe — CHANGES_REQUESTED** [review]
remove excess spaces, otherwise looks good

**Review by MajesticPotatoe — APPROVED** [review]

**Review by MajesticPotatoe — CHANGES_REQUESTED** [review]
fix linting error: 
Disallow self-closing on Vue.js custom components (<v-icon/>)  vue/html-self-closing

**Comment by peterhauke** [comment]
> fix linting error: Disallow self-closing on Vue.js custom components () vue/html-self-closing

OK, done.

**Review by MajesticPotatoe — APPROVED** [review]

**Pull request merged**